recipes/omero-py/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "omero-py" %}
+{% set version = "5.10.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/omero-py-{{ version }}.tar.gz
+  sha256: 4fbf7288586093d45e8f3a661ddf46143f38231017fada39c4e1c26725b34e33
+
+build:
+  entry_points:
+    - omero=omero.main:main
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - appdirs
+    - future
+    - numpy
+    - pillow
+    - python
+    - pywin32  # [win]
+    - pyyaml
+    - requests
+    - zeroc-ice <3.7
+
+test:
+  imports:
+    - omero
+    - omero_ext
+  commands:
+    - pip check
+    - omero --help
+  requires:
+    - pip
+
+about:
+  home: https://github.com/ome/omero-py
+  summary: Python bindings to the OMERO.blitz server
+  license: GPL-2.0-only
```

Comment:
https://github.com/conda-forge/staged-recipes/blob/569b10b8b6fe67c1d2be13ad18c75892bff2b381/recipes/omero-py/meta.yaml#L63-L69 (from https://github.com/conda-forge/staged-recipes/pull/14742)
says `GPL-2.0-or-later`, also might be worth copying the rest of the `about:` metadata?